### DISCUSSION TITLE
AND/OR conditions inside the [BODY]

### DISCUSSION THREAD

**Discussion by dev8ops** [TwiN/gatus#402] [Q&A]
Greetings!
I couldn't find the answer or working syntax myself, so could you clarify?
Does Gatus allow && or || conditions in [BODY] ?
Something like you can do in JsonPath: 
$.Service[0].Foo.HealthChecks[?(@.Component == 'Redis' && @.Status == 'Healthy')]

Thanks in advance!

**Comment by TwiN** [comment]
Gatus' JSONPath implementation does not currently support filters, though there's a feature request open for it: https://github.com/TwiN/gatus/issues/335

As for supporting a logical OR in conditions, either through `||` or through an `or` function, this is definitely something I've been thinking about adding at some point. Feel free to create a feature request for it!
